sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOOrgAssetsLibrary.md

```diff
@@ -25,7 +25,7 @@ Remove-SPOOrgAssetsLibrary [-LibraryUrl <String>] [-ListId <Guid>] [<CommonParam
 
 ## DESCRIPTION
 
-The Remove-SPOOrgAssetsLibrary cmdlet removes a library that was designated as a central location for organization assets across the tenant. Once this cmdlet is run, this library will no longer be accessible from the "Your organization" tab in the file picker. When running the cmdlet, either the library URL or library ID (not both) needs to be indicated.
+The Remove-SPOOrgAssetsLibrary cmdlet removes a library that was designated as a central location for organization assets across the tenant. Once this cmdlet is run, this library will no longer be accessible from the "Your organization" tab in the file picker, (it can take up to 24 hours before the change applies). When running the cmdlet, either the library URL or library ID (not both) needs to be indicated.
```

Comment:
```suggestion
The Remove-SPOOrgAssetsLibrary cmdlet removes a library that was designated as a central location for organization assets across the tenant. Once this cmdlet is run, this library will no longer be accessible from the "Your organization" tab in the file picker (it might take up to 24 hours before the change applies). When running the cmdlet, either the library URL or library ID (not both) needs to be indicated.
```